roles/kube-addons/tasks/main.yml

```diff
@@ -6,9 +6,20 @@
   with_items:
   - kube-system.yml
   - dns.yml
-  - datadog.yml
   - dashboard.yml
 
+- name: deploy kube-datadog
+  include: deploy.yml
+  when: datadog_enabled
+  with_items:
+  - kube-datadog.yml
+
+- name: deploy datadog-agent
+  include: deploy.yml
+  when: (datadog_enabled and datadog_statsd_host == "127.0.0.1:8125")
```

Comment:
hardcoded not template var? 
either way, a comment here why this is might help de-baffle down the road.